README.md

```diff
@@ -18,6 +16,12 @@ Control your Midea M-Smart appliances via local area network.
 
 **_❗Note: Home Assistant 2023.8 or higher required for this integration_**
 
+## Upgrade from georgezhao2010/midea_ac_lan
+
+1. Remove old georgezhao2010/midea_ac_lan integration
+2. [Install current integration](#installation)
+3. Done, your device and data should be exist and works well as before.
```

Comment:
```suggestion
3. Done, your devices, configuration and history data should be there and work as before.
```

Reply:
thank you very much for your review comments, fixed in latest commit